### PR TITLE
Added Heatmap to detect Pokemon Nests

### PR DESCRIPTION
**Pull request by Bitfroest**
## Description
Added Google Maps Heatmap to detect Nests of Pokemon 

## Motivation and Context
Use all the scanned data and show the hotspots of notifiedPokemon.

## How Has This Been Tested?
1. select your Pokemon to show on Heatmap as notifiedPokemon
2. turn on Heatmap
3. you can easily change the heatmap intensity

## Screenshots (if appropriate):
Image shows all captured spots of Karpador
karpador

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: static/dist/js/map.js [hunk 1]
```diff
@@ -28,1220 +30,1303 @@ var pGoStyleNoLabels=[{"featureType":"landscape.man_made","elementType":"geometr
 var selectedStyle = 'light';
 
 var map_data = {
-  pokemons: {},
```
Comment (invisiblek) [inline static/dist/js/map.js]: unecessary whitespace changes throughout this file

**Comment by invisiblek** [comment]
this looks awesome!

**Comment by ingwarsw** [comment]
I would say that feature is really cooll..
I was trying to do same.. but made it looks less awesome..

But you should shqash commits and not reformat all map.js file..

**Comment by invisiblek** [comment]
Merging will squash anyway (the way this repo is set up). But yea, the reformatting needs to be fixed.

**Comment by jangrewe** [comment]
whoa. DO WANT!

**Comment by ingwarsw** [comment]
And PR should go to develp branch?

Yep.. already trying to merge it to mine repo :)

**Comment by invisiblek** [comment]
Ah yep, this pull request should be against develop. We don't push to master.

**Comment by callmebackdraft** [comment]
We do have to keep in mind that Niantic just changed the way nests work, they will migrate.and nests are changing in pokemon types so every now and then the data becomes obsolete

**Comment by invisiblek** [comment]
Yea, probably wouldn't hurt to limit the data set to a few days or a week or something. At some point you should assume the data is stale.

**Comment by billbo99** [comment]
http://www.daterangepicker.com/ is a handy plugin to allow the user to pick date ranges.

**Comment by MRokas** [comment]
Why are you using assign operator in IF at heatmap function? And why the fuck change spaces to tabs..

**Comment by AHAAAAAAA** [comment]
Open a PR against develop. Also, please don't commit credentials.json we just finally got peace again ;-;

**Pull request closed without merging**